### DISCUSSION TITLE
Status Update Request: Unauthorized User Creation and SQL Injection Vulnerability in Chatwoot

### DISCUSSION THREAD

**Discussion by stichtingpleio** [chatwoot/chatwoot#10588] [Q&A]
We previously reported a critical vulnerability in Chatwoot that allows unauthorized users to create accounts. These accounts can then be used to perform an SQL injection, enabling the extraction of all data from the database.

We understand that you are aware of this issue and are working on a fix. Could you please provide us with an update on the status of the resolution and an estimated timeline for the fix?

If the resolution takes too long, we may need to explore the possibility of addressing this issue ourselves.

We appreciate your prompt attention to this matter and look forward to your update.

M Ziemeirnk

**Comment by mazenkhalil** [comment]
Is this related to login with google?

**Comment by pranavrajs** [comment]
@stichtingpleio Missed this thread. Is this related to the filter service? We have added a fix already and have been working on the CVE report. If it is something else, could you please forward an email to security@chatwoot.com.
